### PR TITLE
Added displaying of data in student side

### PR DESCRIPTION
**Pull request by shomzzz**
- displays Student Info in dashboard, grades, and payment assessment
- added condition to display balance status in student information status in dashboard
- added backend functionalities in student side

### CODE REVIEW

File: app/Models/Student.php
Comment (jhaimec) [inline app/Models/Student.php:1]: nilipat yung `pedigree` sa `user` table. Meron din kasi sa faculty yung `pedigree` kaya nalipat sa `user` table yung `pedigree`

File: database/factories/StudentFactory.php
Comment (jhaimec) [inline database/factories/StudentFactory.php:1]: default state ng model?

File: database/migrations/2023_11_09_122608_create_students_table.php
Comment (jhaimec) [inline database/migrations/2023_11_09_122608_create_students_table.php:1]: yung `pedigree` meron na sa `user` table

File: database/seeders/DatabaseSeeder.php
Comment (jhaimec) [inline database/seeders/DatabaseSeeder.php:1]: dito sa `User` probably gawa nlng ng proper dummy account para hindi papalit palit yung data dito. Kahit mema nmn email makakapasok nmn

**Review by jhaimec — COMMENTED** [review]
pa check nlng comments ko

File: database/seeders/DatabaseSeeder.php
Comment (shomzzz) [reply database/seeders/DatabaseSeeder.php:1]: paano palang proper dummy account?

Comment (jhaimec) [reply database/seeders/DatabaseSeeder.php:1]: ano papalit palit ng data ee same lng nmn parang magkakaroon lng ng data na hindi na iaadjust

**Pull request closed without merging**